src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/auth/LdapAuthentication.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hyperdrive.trigger.api.rest.auth
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder
+import org.springframework.security.ldap.authentication.ad.ActiveDirectoryLdapAuthenticationProvider
+import org.springframework.stereotype.Component
+
+@Component
+class LdapAuthentication extends HyperdriverAuthentication {
+
+  @Value("${auth.ad.domain:}")
+  val adDomain: String = ""
```

Comment:
Is the ` = ""` needed? You already specify a default value with the `:` in the `@Value`